tests/unit/lms/views/dashboard/api/course_test.py

```diff
@@ -1,105 +1,32 @@
-import json
 from unittest.mock import sentinel
 
 import pytest
-from h_matchers import Any
 
 from lms.models import Course
-from lms.validation import ValidationError
-from lms.views.dashboard.api.course import CourseViews, ListCoursesSchema
+from lms.views.dashboard.api.course import CourseViews
 from tests import factories
 
 pytestmark = pytest.mark.usefixtures("course_service", "h_api", "organization_service")
 
 
 class TestCourseViews:
-    def test_get_courses(self, course_service, pyramid_request, views, db_session):
```

Comment:
Most of this was testing pagination details. Moved them to pagination focused tests.